madara-prover-rpc-server/src/hints.rs

```diff
@@ -105,6 +105,26 @@ fn unimplemented_hint(
     Ok(())
 }
 
+/*
+Implements hint:
+%{
+    output_start = ids.output_ptr
+%}
+*/
```

Comment:
Use a docstring?

Reply:
Hmmm, I was just following the convention upstream: https://github.com/lambdaclass/cairo-vm/blob/8daae06f4e59dbe7351af54edea4c487a67b3efe/vm/src/hint_processor/builtin_hint_processor/segments.rs#L12C1-L12C1

I'm otherwise unopinionated though

Reply:
Fair enough.